### PR TITLE
Fixes for grammar

### PR DESCRIPTION
**Pull request by waterweasel4**
# Pull Request

Thank you for your contribution to the Bulwark. Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
N/A

## Type of change

- [ ] A new feature
- [x] A bug fix
- [ ] Documentation only changes
- [x] Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
- [ ] A code change that neither fixes a bug nor adds a feature
- [ ] A code change that improves performance
- [ ] Adding missing tests or correcting existing tests
- [ ] Changes that affect the build system or external dependencies
- [ ] Changes to our CI configuration files and scripts
- [ ] Other changes that don't modify src or test files

Description

```
When an assessment lasts only one day, the report was showing "the assessment last for 1 days" (plural).  This has been changed such that if day = 0, or day > 1, report 'days'.  Otherwise, report 'day'.
```

---

## Checklist

- [x] My code follows the [contributing](../CONTRIBUTING.md) guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,6 +82,7 @@
     "class-validator": "^0.12.2",
     "concurrently": "^5.3.0",
     "cors": "^2.8.5",
+    "cz": "^1.8.2",
```
Comment (Whamo12) [inline package.json:85]: Is this package needed?

File: frontend/src/app/report/report.component.ts [hunk 2]
```diff
@@ -37,6 +38,12 @@ export class ReportComponent implements OnInit {
           Date.parse(this.report.assessment.startDate)) /
           86400000
       );
+      if (this.numOfDays > 1 || this.numOfDays  === 0) {
```
Comment (Whamo12) [inline frontend/src/app/report/report.component.ts:41]: This would be a good opportunity to use the ternary operator.

**Review by Whamo12 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Whamo12) [reply package.json:85]: Ah, I get it. You most likely did not have the package installed locally to run the 'npm run commit' script. Go ahead and update the 'npm run commit' script to use npx which allows you to run the script without package installation. 

'npx git-cz'

**Review by waterweasel4 — COMMENTED** [review]
Added ternary operator and removed my package.json edit.

**Review by Whamo12 — APPROVED** [review]

**Pull request merged**